### ISSUE TITLE
IJulia on Win10

### ISSUE THREAD

**Issue by bkamins** [JuliaLang/IJulia.jl#735]
The newest release of IJulia is much more stable - thank you! However, when running complex notebooks like https://github.com/bkamins/Julia-DataFrames-Tutorial/blob/master/13_extras.ipynb it still occasionally hangs on Win10.

What debug info would be most helpful to diagnose the reason?

**Comment by stevengj** [comment]
It's hard to do much unless you can boil it down to a minimal example.  (It's not even clear yet that the origin of the problem is in IJulia.)

**Comment by bkamins** [comment]
Thank you. I will try to distill down the source of the problem and report back with the details.

But for now the short story is:
* if I extract source code to a `.jl` file and remove lines that throw an error the file runs without a problem;
* I managed to evaluate the whole notebook if I run it in chunks (e.g. run section 1, save, close Jupyter, repeat for next sections);
* however, if you e.g. select "Run all" then I have managed to run it all without hanging.
